lib/end.go

```diff
@@ -29,14 +29,14 @@ import (
 // End will end the build, storing the untarred ACI stored at tmpaci into
 // output. The files at contextpath will be removed, to end the build. If
 // overwrite is true, an error will not be thrown if output already exists.
-func End(tmpaci, output, contextpath string, overwrite bool) error {
+func End(tmpaci, output, contextpath string, overwrite, sign bool, gpgflags []string) error {
 	man, err := util.GetManifest(tmpaci)
 	if err != nil {
 		return err
 	}
 
 	if man.App != nil && testEq(man.App.Exec, placeholderexec) {
-		return fmt.Errorf("can't end build, run command was never set")
+		return fmt.Errorf("can't end build, exec command was never set")
```

Comment:
Welp, this is no longer valid thanks to https://github.com/appc/spec/pull/505. Although I would still suggest we at leats print a warning in this case.